packages/docs/src/pages/components/switch.mdx

```diff
@@ -10,10 +10,65 @@ Form switch input component
 import { Switch } from 'theme-ui'
 ```
 
-```js live=true
+```jsx live=true
 <Switch label="Enable email alerts?" />
 ```
 
+## Input state style `&:checked`
+
+The Switch component renders several elements, among other things an
+`<input type="checkbox" />` which state can be used to update the style.
+
+However, since this input is hidden by CSS (the "visual switch" is a Box),
+applying for instance the `&:checked` CSS selector will not show any difference.
+
+To show different style based on the input state, you can use
+[Emotion's `&` nest selector](https://emotion.sh/docs/styled#nesting-components)
```

Comment:
I'm not sure if "nest selector" is the right word here. I've been always reading it as _self_ or _this_.

Though, it seems people usually just call it "ampersand" — https://css-tricks.com/the-sass-ampersand/.

Reply:
My bad, misread Emotion's doc and thought they actually called it `nest selector`

What about just `Emotion's & selector` (without naming it?)

Reply:
+1